.circleci/config.yml

```diff
@@ -164,15 +176,15 @@ jobs:
       - checkout
       - run:
           name: Open simulator
-          command: xcrun instruments -w "iPhone 11 Pro (14.5) [" || true
+          command: xcrun instruments -w "iPhone 11 Pro (15.0) [" || true
```

Comment:
turns out this happens because I removed the `arch(64)` check and the workaround suggested by Xcode's known issues section in the release notes (which was to update the deployment target to 12.0) doesn't work correctly. 
I _think_ it did work in beta 5? Or maybe in apple silicon? Will test both out for science.